core/src/main/kotlin/org/evomaster/core/mongo/MongoDbAction.kt

```diff
@@ -0,0 +1,56 @@
+package org.evomaster.core.mongo
+
+import org.evomaster.core.search.Action
+import org.evomaster.core.search.gene.Gene
+import org.evomaster.core.search.gene.ObjectGene
+import java.lang.reflect.Modifier
+import java.util.*
+
+class MongoDbAction(
```

Comment:
i will likely do some refactoring, to enable/simply having different kinds of initializing actions. might do on a branch, or here, i will see, hopefully this week

Reply:
Great!